openghg/standardise/_standardise.py

```diff
@@ -40,6 +40,24 @@ def standardise(data_type: str, filepath: multiPathType, store: Optional[str] =
         logger.info("Compression disabled")
         compressor = None
 
+    if data_type == "surface" and kwargs["source_format"].lower() != "gcwerks":
```

Comment:
A few things on this:
1. If we're only doing this for specific data types could we move this to within the `standardise_surface` and `standardise_footprint` functions instead? Would be nice to keep `standardise` as generic as possible.
2. Since this code is repeated, would be good to create a util function for this to simplify these lines.